tests/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -0,0 +1,178 @@
+            absorption_data_sets={},
+        )
+
+        surface = eradiate.scenes.surface.LambertianSurface(
+            reflectance=reflectance,
+        )
+
+        return eradiate.experiments.OneDimExperiment(
+            measures=[measure],
+            illumination=illumination,
+            atmosphere=atmosphere,
+            surface=surface,
+        )
+
+    def configure_ckd_experiment(
+        bin_set,
+        bins,
+        spp=32,
+        reflectance=0.5,
+        zenith=30.0,
+        n_vza=90,
+    ):
+        eradiate.set_mode("ckd")
+        illumination = eradiate.scenes.illumination.DirectionalIllumination(
+            zenith=zenith, azimuth=0.0
+        )
+
+        atmosphere = eradiate.scenes.atmosphere.MolecularAtmosphere.afgl_1986(
+            id="molecules",
+            has_absorption=True,
+            has_scattering=True,
+        )
+
+        surface = eradiate.scenes.surface.LambertianSurface(
+            reflectance=reflectance,
+        )
+
+        measure = {
+            "type": "distant",
+            "construct": "from_viewing_angles",
+            "spectral_cfg": eradiate.scenes.measure.MeasureSpectralConfig.new(
+                bin_set=bin_set,
+                bins=bins,
+            ),
+            "zeniths": np.linspace(-90, 90, n_vza) * ureg.deg,
+            "azimuths": 0.0 * ureg.deg,
+            "spp": spp,
+        }
+
+        return eradiate.experiments.OneDimExperiment(
+            measures=[measure],
+            illumination=illumination,
+            atmosphere=atmosphere,
+            surface=surface,
+        )
+
+    # Settings
+    zenith = 50.0 * ureg.deg
+    reflectance = 0.0 * ureg.dimensionless
+    n_vza = 40
+    spp = 1e5
+    wavelengths = np.linspace(545.0, 555.0, 2) * ureg.nm  # mono mode setting
+    bin_set = "10nm"  # ckd mode setting
+    bins = "550"  # ckd mode setting
+
+    # Monochromatic experiment
+    mono_exp = configure_mono_experiment(
+        wavelengths=wavelengths,
+        spp=spp,
+        reflectance=reflectance,
+        zenith=zenith,
+        n_vza=n_vza,
+    )
+    with uck.override(length="km"):  # this is a temporary workaround the 'batman' issue
+        mono_exp.run()
+    mono_results = mono_exp.results["measure"]
+    mono_results_averaged = mono_results.mean(dim="w")
+
+    # CKD experiment
+    ckd_exp = configure_ckd_experiment(
+        bin_set=bin_set,
+        bins=bins,
+        spp=spp,
+        reflectance=reflectance,
+        zenith=zenith,
+        n_vza=n_vza,
+    )
+    ckd_exp.run()
+    ckd_results = ckd_exp.results["measure"]
+
+    # Make figure
+    fig = plt.figure(figsize=(8, 5))
+    filename = f"test_onedim_ckd_vs_mono.png"
+    ensure_output_dir(os.path.join(output_dir, "plots"))
+    fname_plot = os.path.join(output_dir, "plots", filename)
+
+    mono_results_averaged.brf.squeeze().where(mono_results_averaged.brf.vza < 75).where(
+        mono_results_averaged.brf.vza > -75
```

Comment:
I think the following is more elegant and should work (please double-check):
```suggestion
    mono_results_averaged.brf.squeeze().where(
        np.logical_and(
            mono_results_averaged.brf.vza < 75, mono_results_averaged.brf.vza > -75
        )
    )
```

Reply:
With the vza parametrisation that you suggested, this is not necessary anymore :)